camera_separation: 0.37
focal_distance: 525
